x-team/auto-ui
# Update ActivityLogsDisplay

**Release Type:** *Non-Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-2665

## Description

Added a value variation for grouped activities. Basically the prop `groupValues` at `<ActivityLog/>`.

Another addition was the element `info` in the activity data that helps display further info beyond the original value of each element in grouped displays.

In this PR also have some extra stories for that component.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-2665-update-grouping-of-lists-accept-exclude-activity-logs-items` | [Auto](https://github.com/x-team/auto/pull/1776)

## Steps to Test or Reproduce

1. On this branch run `npm start`
1. Navigate to http://localhost:9001/?selectedKind=UI%20Components%2FActivityLogsDisplay&selectedStory=complete%20with%20random%20data%20with%20knobs (see screenshot bellow)
    1. Observe that `ActivityLogsDisplay` behaves properly with some sort of content variation
    1. Observe that `ActivityLog` behaves properly with some sort of content variation

## Impacted Areas in Application

`ActivityLogsDisplay` and `ActivityLog`

## Screenshots

<img width="1680" alt="screen shot 2019-01-17 at 19 07 54" src="https://user-images.githubusercontent.com/131859/51349377-56789f00-1a8c-11e9-97f6-352732c67c01.png">
